app/controllers/ctc/portal/portal_controller.rb

```diff
@@ -16,15 +16,18 @@ def home
     end
   end
 
-  def edit_info; end
+  def edit_info
+    @current_submission = current_intake.client.efile_submissions.last
+  end
 
   def resubmit
     @submission = current_client.efile_submissions.last
```

Comment:
In theory, there's a race condition here. A new EfileSubmission could be created between when the page is loaded vs. when the button is clicked.

Even though it's pedantic, I'd personally prefer if we sent the efile_submission ID to the server. Then we can detect this race and show an error message or something.

Not a blocker.